src/Nethermind/Nethermind.JsonRpc/ErrorType.cs

```diff
@@ -89,5 +89,10 @@ public static class ErrorCodes
         /// 
         /// </summary>
         public const int ExecutionError = -32015;
+         
+        /// <summary>
+        /// Request exceedes defined tracer timeout limit
+        /// </sumary>
+        public const int TracerTimeout = -32016;
```

Comment:
what is the number thrown by Geth?

Reply:
Geth doesn't have JsonRpc error code for timeout 
[Geth's jsonRpc errors](https://github.com/ethereum/go-ethereum/blob/beabf95ad7fd80db83944400348c1e42ac3b8c35/rpc/errors.go#L34)

it have timeouts for http client so I guess they just throw 408 in response 
[Geth's HttpTimeout](https://github.com/ethereum/go-ethereum/blob/master/rpc/http.go#L71)